starboard/nplb/posix_compliance/posix_pipe_test.cc

```diff
@@ -172,16 +185,36 @@ TEST(PosixPipeTest, ReadFromEmptyPipeCreatedWithNonBlockFlagDoesNotBlock) {
   close(pipe_fds[1]);
 }
 
-void* DoDelayedRead(void* parameter) {
+void* DoReadOnTimeout(void* parameter) {
```

Comment:
Can we rename the parameter here to i.e. `test_context` (or `context` and rename the local var below to `test_context`)? `parameter` as a name is too generic